### ISSUE TITLE
Along with Images

### ISSUE THREAD

**Issue by tekjar** [Grayda/sphere-documentation#1]
Let's try to include lot of images with USER documentation.

for example in --> https://github.com/Grayda/sphere-documentation/wiki/LED-Images-and-their-meaning
Is it possible to include images in .md files ?

**Comment by theshaun** [comment]
Certainly, the markdown documentation explains it [here](https://guides.github.com/features/mastering-markdown/#examples)

Simply provide the image with alternative text like this

```
![cat](https://media3.giphy.com/media/tuvMgAPzxaQBq/200_s.gif)
```

![cat](https://media3.giphy.com/media/tuvMgAPzxaQBq/200_s.gif)

**Comment by Grayda** [comment]
Also, if we're going to include screenshots, we should also note down any relevant text that people might want to copy (e.g. if we're showing a command to type in, or something people might want to search on Google, include the text as a caption). I've read far too many tutorials that have great content, but the stuff you want to Google to research, is in a tiny screenshot

Also, some of the LED matrix icons can be extracted from GitHub or the Sphere itself (in the sphere-led-controller repo, I think). They're 16x16, but blown up, they look sort of like what you'd see on top of the Sphere

**Comment by paulcull** [comment]
would be helpfull to get a library of 16x16 images

**Comment by Grayda** [comment]
The images are stored [here](https://github.com/ninjasphere/sphere-go-led-controller/tree/master/ninjapack/root/opt/ninjablocks/drivers/sphere-go-led-controller/images). To get them to display nicely they need to be resized without whatever editing program blurring them.

EDIT: And just for anyone reading, icons can be loaded from /data/icons. This is more for anyone tinkering with the source and adding their own icons to apps. But still good to know.

**Comment by theshaun** [comment]
http://ezgif.com/resize lets you resize them without loosing 'quality' 
![ezgif-1207592179](https://cloud.githubusercontent.com/assets/712995/7692773/640f3fbc-fe0b-11e4-8116-621e4b87b28a.gif)

They kinda look pretty cool

**Comment by Grayda** [comment]
I just tried a few and they do look pretty good. The `Gifsicle` setting works best for sharper edges than Imagemagick which seems to blend the edges for a smoother look

![Heater On GIF](http://im.ezgif.com/tmp/ezgif-2581216988.gif)

EDIT: I've added some basic instructions to the wiki for adding LED matrix gifs
